composer.json

```diff
@@ -16,7 +16,7 @@
     }
   },
   "require": {
-    "microsoft/kiota-abstractions": "^0.4.0",
+    "microsoft/kiota-abstractions": "^0.5.0",
```

Comment:
```suggestion
    "microsoft/kiota-abstractions": "^0.6.0",
```

Reply:
It's the same issue. I created a PR for it.